src/amo/components/PermissionsCard/styles.scss

```diff
@@ -0,0 +1,29 @@
+.PermissionCard-learn-more {
+  margin: 12px auto 0;
+  width: 100%;
+
+  .Icon {
+    margin-left: 10px;
```

Comment:
I see `11px` in the mocks but since we try to adhere to multiples of 4 (the mocks aren't always exact), I'd suggest `12px`.